src/main/java/me/botsko/prism/monitors/UseMonitor.java

```diff
@@ -1,179 +1,127 @@
 package me.botsko.prism.monitors;
 
-import java.util.List;
-import java.util.concurrent.ConcurrentHashMap;
-
+import me.botsko.prism.Prism;
 import me.botsko.prism.utils.MiscUtils;
 import me.botsko.prism.utils.TypeUtils;
 import org.bukkit.ChatColor;
 import org.bukkit.block.Block;
 import org.bukkit.entity.Player;
 
-import me.botsko.prism.Prism;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
 
 public class UseMonitor {
+    protected final List<String> blocksToAlertOnPlace;
+    protected final List<String> blocksToAlertOnBreak;
+    private final Prism plugin;
+    private ConcurrentHashMap<String, Integer> countedEvents = new ConcurrentHashMap<>();
+    /**
+     * Constructor
+     * @param plugin Prism
+     */
+    public UseMonitor(Prism plugin) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['CTOR_DEF' should be separated from previous statement.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)